README.md

```diff
@@ -101,6 +101,15 @@ Meilisearch is a search engine created by [Meili](https://www.welcometothejungle
 
 Thank you for your support!
 
-## 📦 Internal crates and their versioning
+## 👩‍💻 Contributing
 
-The crates in this repository are not currently available on crates.io and do not follow [semver conventions](https://semver.org). However, the Meilisearch search engine is well versioned, and releases follow the semver conventions.
+Meilisearch is and always will be open-source! If you want to contribute to the project, please take a look at this [guidelines](CONTRIBUTING.md).
+
+We are looking forward to review your contribution! ❤️
```

Comment:
I feel this doesn't add much to what has already been said. But it boils down to personal preference. Feel free to push back.

```suggestion
```